DBMS/DBMS-Assignments/DBMS-Assignment-3/Assignment-1.sql

```diff
@@ -0,0 +1,91 @@
+use store_front ;
+
+#Q1  With respect to StoreFront application identify, apply and list the constraints you would apply on the columns for the tables created.
+
+
+CREATE TABLE user_details (
+user_id INT NOT NULL AUTO_INCREMENT,
+password varchar(12) NOT NULL,
+PRIMARY KEY (user_id)
+);
+
+CREATE TABLE shopper (
+shopper_id INT NOT NULL,
+shopper_name varchar(20) NOT NULL,
+mobile BIGINT (10) NOT NULL,
+FOREIGN KEY (shopper_id) REFERENCES user_details(user_id)
+);
+
+CREATE TABLE admin_details (
+admin_id INT ,
+admin_name VARCHAR(20) NOT NULL ,
+email VARCHAR(30) NOT NULL ,
+FOREIGN KEY (admin_id) REFERENCES user_details(user_id)
+);
+
+CREATE TABLE shopper_address (
+shopper_id INT,
+address VARCHAR(50) NOT NULL,
+FOREIGN KEY (shopper_id) REFERENCES user_details(user_id)
+);
+
+CREATE TABLE category(
+category_id INT AUTO_INCREMENT,
+category_name VARCHAR(20) NOT NULL ,
```

Comment:
Change it to `name` as it is category table columns, so the name would be category name.